pkg/operator/controllers/dnsmasq/machineconfig_controller.go

```diff
@@ -55,6 +55,10 @@ func (r *MachineConfigReconciler) Reconcile(ctx context.Context, request ctrl.Re
 		return reconcile.Result{}, nil
 	}
 
+	if instance.Spec.OperatorFlags.GetSimpleBoolean(restartDnsmasqEnabled) {
```

Comment:
nit: Can we assign the operator flag's value to the intermediate variable `restartDnsmasq` and use it both here and in the call to `reconcileMachineConfigs`? Just so the three controllers that reference the flag match each other in this way.

Reply:
Yes, I hadn't really thought about it that way. I choice not to use a variable here simply because it wasn't necessary. But I like your suggestion of continuity, I'll make a follow up PR.